pykickstart/commands/authconfig.py

```diff
@@ -22,7 +22,7 @@
 
 from pykickstart.errors import KickstartDeprecationWarning
 from pykickstart.version import FC3, versionToLongString, F28
```

Comment:
You need to import F35.

Also, you can run `make check` and `make test` locally to check the tests before pushing.

Reply:
This should be fixed now. Make check yields many errors like:
```
************* Module /home/pbrezina/workspace/pykickstart/tests/commands/network.py
E1102(not-callable):116,14: FC3_TestCase.runTest: self.handler is not callable
```

Maybe I run it wrong?

Reply:
Possibly a different pylint version? I'm running on Fedora 33 with python3-pylint-2.6.0-2.fc33.noarch and don't see it complain about that. Running with tox also doesn't return that error so I'm not sure what's going on with your setup.